test/evergreen.yml

```diff
@@ -291,13 +291,19 @@ functions:
         export WIREDTIGER_CONFIG='checkpoint_sync=0,transaction_sync=(method=none)'
         CMD='./t -h WT_TEST.$i.$t -t r -r 2 -W 3 -n 1000000 -k 1000000 -C "cache_size=100MB"'
 
+        ret=-1
```

Comment:
Any special reason the default return value is set to `-1` (instead of e.g. 0)?
Suggest to move this line above the for loop that uses the `ret` variable.

Reply:
I'm a bit confused by this as well, along with the `if [ $ret -ne -1 ]` check. What is the scenario where `ret=-1`?

Reply:
Fixed. Initialized ret to 0 and checking if ret has some other value (during failure).